src/P/Function.hs

```diff
@@ -1,12 +1,26 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 module P.Function (
     applyN
+  -- * Functions for development/debuggung only
+  -- | Cannot be used in production code, but might be useful
+  -- during development or debugging
+  , undefined
+  , error
```

Comment:
It might be better to put these in a separate module like `P.Development` that isnt re-exported by `P`?

this will also mean that the branch build for WIP will also fail while these are in place, not sure if thats a big problem or not